platform/core/PolyPodCoreSwift/Sources/PolyPodCoreSwift/UpdateNotification.swift

```diff
@@ -0,0 +1,43 @@
+import Foundation
+
+public class UpdateNotification {
```

Comment:
This can be an enum, so it can never be constructed; so it simply acts as namespace.